setup.py

```diff
@@ -23,6 +23,7 @@
 
 requirements = [
     "qiskit>=0.25",
+    "mthree>=0.17",
```

Comment:
It is not included in Qiskit installation but it is the only outside package that can be imported in runtime. Technically we don't need it but I was getting import failures if not installed. Any suggestions?

Reply:
> Technically we don't need it but I was getting import failures if not installed.

Oh really? Is it a hard requirement or it's a possibility to use it within a Runtime program?

If it's optional, then we should treat it as for example we treat TensorFlow in PennyLane core: it's not a hard requirement, but if the user wants to access features that depend on it then an error is raised. This will help prevent cases where potential changes to `mthree` might influence the entire plugin.